tools/util/gtf2featureAnnotation.xml

```diff
@@ -0,0 +1,74 @@
+<tool id="_ensembl_gtf2feature_annotation" name="GTF2GeneAnnotationTable" version="1.42.1+galaxy0">
+    <description>extracts a complete annotation table or subsets thereof from an Ensembl GTF using rtracklayer</description>
+    <requirements>
+      <requirement type="package" version="1.42.1">bioconductor-rtracklayer</requirement>
+      <requirement type="package">r-optparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       ln -s '$gtf_input' input.gtf;
+       $__tool_directory__/gtf2featureAnnotation.R --gtf-file input.gtf
+#if $noheader
+--no-header
+#end if
+#if $mito.mark_mito
+--mito --mito-chr "${mito.mito_chr}" --mito-biotypes "${mito.mito_biotypes}" 
+#end if
+--feature-type "${feature_type}" --first-field "${first_field}" --output-file annotation.txt --fields "${fields}"
+	    ]]></command>
+
+    <inputs>
+        <param name="gtf_input" type="data" format="gff" label="Ensembl GTF file" />
+        <param name="feature_type" type="text" optional='true' value="gene" label="Feature type for which to derive annotation"/>
+        <param name="first_field" type="text" optional='true' value="gene_id" label="Field to place first in output table"/>
+        <param name="noheader" type="boolean" checked="false" label="Suppress header line in output?"/>
+        <param name="fields" type="text" optional='true' value="" label="Comma-separated list of field names to extract from the GTF (default: use all fields)"/>
+        <conditional name="mito">
+          <param name="mark_mito" type="boolean" checked="true" label="Flag mitochondrial features?"/>
+          <when value="true">
+            <param name="mito_chr" type="text" optional='true' value="mt,mitochondrion_genome,mito" label="Comma-separated list of possible mitochondrial chromosome names (case insensitive)"/>
```

Comment:
Perhaps change the default value to something like "M,MT,chrM,chrMt"?

Reply:
So, it's case insensitive, and these are the chromosome names our existing annotations have for mitochondria. But it can be fairly exhaustive- I'll include m, chrm and chrMt as well.